packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -1,10 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+
 /**
  * Internal dependencies
  */
 import useClipboardHandler from '../writing-flow/use-clipboard-handler';
 
+/**
+ * @deprecated
+ */
+export const __unstableUseClipboardHandler = () => {
+	deprecated( '__unstableUseClipboardHandler', {
+		alternative: 'BlockCanvas or WritingFlow',
+		version: '6.4',
```

Comment:
I think that's the version it's supposed to be removed in. So maybe this should be "since" instead of "version" or use a higher version number like 6.7 or 6.8

Reply:
Oh!